utils/src/main/java/org/robolectric/util/Strftime.java

```diff
@@ -19,7 +20,7 @@ public class Strftime {
    * @param zone The timezone to use for formatting.
    * @return The formatted datetime.
    */
-  public static String format(String format, final Date date, Locale locale, TimeZone zone) {
+  public static String format(String format, final Instant date, Locale locale, TimeZone zone) {
     StringBuilder buffer = new StringBuilder();
```

Comment:
We can't change this method's signature, because it has been released to users.

Reply:
Okay, changed it back and suppressed the warning